docs/root/intro/arch_overview/dynamic_configuration.rst

```diff
@@ -27,7 +27,7 @@ via the built in :ref:`hot restart <arch_overview_hot_restart>` mechanism.
 Though simplistic, fairly complicated deployments can be created using static configurations and
 graceful hot restarts.
 
-.. _arch_overview_dynamic_config_sds:
+.. _arch_overview_dynamic_config_eds:
```

Comment:
As long as you are in here, do you mind just purging SDS from this file (in relation to EDS), switching to EDS, and then maybe having a section on SDS (secrets at the end). We are turning down V1 anyway and I have been meaning to do this. cc @alyssawilk

Reply:
Oh, yes, I will certainly do that! I didn't realize things were that far along in moving on from v1.

Reply:
Done; PTAL.